README.md

```diff
@@ -14,18 +14,42 @@ This script aims to automate the process of initializing a React App using the W
 
 ## How to use
 
+To run from server:
+
 You don't need to clone this repository. Just follow these steps:
 
 1- Run the installation script using the following command:
 
 ```bash
-bash <(curl -s https://raw.githubusercontent.com/Wolox/react-bootstrap/development/run.sh) [--verbose]
+bash <(curl -s https://raw.githubusercontent.com/Wolox/react-bootstrap/development/run.sh)
 ```
+Optional parameters:
+
+- `-v` or `--verbose`: Display more information as the bootstrap is running
+- `-l` or `--local`: Run a local version of the bootstrap
 
 2- Search for `TODO` commented lines and follow the instructions. (Search "// TODO")
```

Comment:
Sorry, I just realized that we should remove the 2) and 3), as they're no longer necessary. And remove the number (1) from the first point:
`Run the installation script using the following command:`